scalableminds/webknossos
# Per-User LRU sorting of Datasets

### URL of deployed dev instance (used for testing):
- https://lrudatasets.webknossos.xyz

### Steps to test:
- click on dataset
- after that, navigate to dataset list view, with browser network tab open
- in list reply, datasets should have `lastUsedByUser` field, non-zero for the one you clicked on before

- the sorting itself is disabled for now, to first gather some data, see comment below

### Issues:
- fixes #2917

------
